File: C++/Mahasiswa.cpp
#include <bits/stdc++.h>
#include "SivitasAkademik.cpp"
using namespace std;
class Mahasiswa : public SivitasAkademik
{
private:
    string NIM, fakultas, prodi;

public:
    Mahasiswa()
    {
        this->NIM = "";
        this->fakultas = "";
        this->prodi = "";
    };
    Mahasiswa(string NIM, string fakultas, string prodi)
    {
        this->NIM = NIM;
        this->fakultas = fakultas;
        this->prodi = prodi;
    }
    void setNIM(string NIM)
    {
        this->NIM = NIM;
    }
    void setFakultas(string fakultas)
    {
        this->fakultas = fakultas;
    }
    void setProdi(string prodi)
    {
        this->prodi = prodi;
    }
    string getNIM() { return this->NIM; }
    string getFakultas() { return this->fakultas; }
    string getProdi() { return this->prodi; }
    ~Mahasiswa(){

    };
};

File: PHP/program/Mahasiswa.php
<?php
require("SivitasAkademik.php");
class Mahasiswa extends SivitasAkademik
{
    private $NIM;
    private $prodi;
    private $fakultas;

    public function __construct($NIM = "", $prodi = "", $fakultas = "")
    {
        $this->NIM = $NIM;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }
    public function setNIM($NIM)
    {
        $this->NIM = $NIM;
    }
    public function setProdi($prodi)
    {
        $this->prodi = $prodi;
    }
    public function setFakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }
    public function getNIM()
    {
        return $this->NIM;
    }
    public function getProdi()
    {
        return $this->prodi;
    }
    public function getFakultas()
    {
        return $this->fakultas;
    }
}
